docs/core/diagnostics/installing.md

```diff
@@ -0,0 +1,60 @@
+---
```

Comment:
This article seems a duplication of what we have in https://docs.microsoft.com/en-us/dotnet/core/tools/global-tools#install-a-global-tool and https://docs.microsoft.com/en-us/dotnet/core/tools/troubleshoot-usage-issues. 

If there's anything missing, we should add to those articles instead.

My recommendation would be to remove this one.